homeassistant/components/emoncms/strings.json

```diff
@@ -35,6 +38,10 @@
     "missing_include_only_feed_id": {
       "title": "No feed synchronized with the {domain} sensor",
       "description": "Configuring {domain} using YAML is being removed.\n\nPlease add manually the feeds you want to synchronize with the `configure` button of the integration."
+    },
+    "migrate_database": {
+      "title": "Upgrade your emoncms version",
+      "description": "Your [emoncms]({url}) does not ship a unique identifier.\n\n Please upgrade to at least version 11.5.7 and migrate your emoncms database.\n\n More info on [emoncms documentation](https://docs.openenergymonitor.org/emoncms/update.html#upgrading-to-a-version-producing-a-unique-identifier)"
```

Comment:
Please use a placeholder for the documentation URL

Reply:
OK done